Description: Patch away is-ci and use ci-info instead
Author: Paolo Greppi <paolo.greppi@libpf.com>
Forwarded: not-needed
Reviewed-By: Yadd <yadd@debian.org>
Last-Update: 2020-11-07

--- a/packages/jest-cli/src/cli/args.ts
+++ b/packages/jest-cli/src/cli/args.ts
@@ -5,7 +5,7 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import isCI = require('is-ci');
+import {isCI} from 'ci-info';
 import type {Config} from '@jest/types';
 import {constants, isJSONString} from 'jest-config';
 
--- a/packages/jest-util/src/isInteractive.ts
+++ b/packages/jest-util/src/isInteractive.ts
@@ -5,6 +5,6 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import isCI = require('is-ci');
+import {isCI} from 'ci-info';
 
 export default !!process.stdout.isTTY && process.env.TERM !== 'dumb' && !isCI;
